### ISSUE TITLE
[WARNING STA-0349] instance 'ces_0_0' not found.

### ISSUE THREAD

**Issue by oharboe** [The-OpenROAD-Project/OpenROAD#5383]
### Describe the bug

Based on https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/2120/commits/113fd8dd5c43002b92661c0ff6fcb882e286f809

1. untar [power.tar.gz](https://github.com/user-attachments/files/16243517/power.tar.gz)
2. run `./run-me-mock-array-asap7-base.sh`

Output:

```
OpenROAD v2.0-14511-gedb4e7e37 
Features included (+) or not (-): +Charts +GPU +GUI +Python
This program is licensed under the BSD-3 license. See the LICENSE file for details.
Components of this program may be licensed under more restrictive licenses which must be honored.
[INFO ODB-0227] LEF file: /home/oyvind/OpenROAD-flow-scripts/flow/platforms/asap7/lef/asap7_tech_1x_201209.lef, created 24 layers, 9 vias
[INFO ODB-0227] LEF file: /home/oyvind/OpenROAD-flow-scripts/flow/platforms/asap7/lef/asap7sc7p5t_28_R_1x_220121a.lef, created 212 library cells
Reading results/asap7/mock-array_Element/base/6_final.v
Reading ./results/asap7/mock-array/base/6_final.v
[WARNING ORD-2011] LEF master DECAPx10_ASAP7_75t_R has no liberty cell.
[WARNING ORD-2011] LEF master DECAPx6_ASAP7_75t_R has no liberty cell.
[WARNING ORD-2011] LEF master FILLERxp5_ASAP7_75t_R has no liberty cell.
[WARNING ORD-2011] LEF master FILLER_ASAP7_75t_R has no liberty cell.
[WARNING ORD-2011] LEF master DECAPx1_ASAP7_75t_R has no liberty cell.
[WARNING ORD-2011] LEF master DECAPx2_ASAP7_75t_R has no liberty cell.
[WARNING ORD-2011] LEF master DECAPx4_ASAP7_75t_R has no liberty cell.
[WARNING ORD-2011] LEF master TAPCELL_ASAP7_75t_R has no liberty cell.
[WARNING STA-0349] instance 'ces_0_0' not found.
[WARNING STA-0349] instance 'ces_0_1' not found.
[WARNING STA-0349] instance 'ces_0_2' not found.
[deleted]
```

### Expected Behavior

No warnings and able to run read_power_activities from .vcd file.

### Environment

```shell
OpenROAD v2.0-14511-gedb4e7e37
```

### To Reproduce

See above

### Relevant log output

```
OpenROAD v2.0-14511-gedb4e7e37 
Features included (+) or not (-): +Charts +GPU +GUI +Python
This program is licensed under the BSD-3 license. See the LICENSE file for details.
Components of this program may be licensed under more restrictive licenses which must be honored.
[INFO ODB-0227] LEF file: /home/oyvind/OpenROAD-flow-scripts/flow/platforms/asap7/lef/asap7_tech_1x_201209.lef, created 24 layers, 9 vias
[INFO ODB-0227] LEF file: /home/oyvind/OpenROAD-flow-scripts/flow/platforms/asap7/lef/asap7sc7p5t_28_R_1x_220121a.lef, created 212 library cells
Reading ./results/asap7/mock-array/base/6_final.v
Reading results/asap7/mock-array_Element/base/6_final.v
Linking
[WARNING ORD-2011] LEF master DECAPx10_ASAP7_75t_R has no liberty cell.
[WARNING ORD-2011] LEF master DECAPx6_ASAP7_75t_R has no liberty cell.
[WARNING ORD-2011] LEF master FILLERxp5_ASAP7_75t_R has no liberty cell.
[WARNING ORD-2011] LEF master FILLER_ASAP7_75t_R has no liberty cell.
[WARNING ORD-2011] LEF master DECAPx1_ASAP7_75t_R has no liberty cell.
[WARNING ORD-2011] LEF master DECAPx2_ASAP7_75t_R has no liberty cell.
[WARNING ORD-2011] LEF master DECAPx4_ASAP7_75t_R has no liberty cell.
[WARNING ORD-2011] LEF master TAPCELL_ASAP7_75t_R has no liberty cell.
Read SDC
[WARNING STA-0349] instance 'ces_0_0' not found.
[WARNING STA-0349] instance 'ces_0_1' not found.
[deleted]
[WARNING STA-0363] pin 'ces_0_0/io_lsbOuts_0' not found.
[WARNING STA-0363] pin 'ces_0_0/io_lsbOuts_1' not found.
[WARNING STA-0363] pin 'ces_0_0/io_lsbOuts_2' not found.
[deleted]
[WARNING STA-0363] message limit reached, this message will no longer print
[WARNING STA-1647] ./results/asap7/mock-array/base/6_final.spef line 86052, pin ces_0_0 not found.
[WARNING STA-1647] ./results/asap7/mock-array/base/6_final.spef line 86053, pin ces_1_0 not found.
[deleted]
[WARNING STA-1647] message limit reached, this message will no longer print
Group                  Internal  Switching    Leakage      Total
                          Power      Power      Power      Power (Watts)
----------------------------------------------------------------
Sequential             1.99e-01   6.59e-04   4.96e-06   1.99e-01  61.0%
Combinational          3.29e-04   1.38e-04   5.93e-06   4.72e-04   0.1%
Clock                  6.92e-02   5.77e-02   6.48e-07   1.27e-01  38.8%
Macro                  0.00e+00   0.00e+00   0.00e+00   0.00e+00   0.0%
Pad                    0.00e+00   0.00e+00   0.00e+00   0.00e+00   0.0%
----------------------------------------------------------------
Total                  2.68e-01   5.85e-02   1.15e-05   3.27e-01 100.0%
                          82.1%      17.9%       0.0%
Annotated 183622 pin activities.
Group                  Internal  Switching    Leakage      Total
                          Power      Power      Power      Power (Watts)
----------------------------------------------------------------
Sequential             1.96e-01   0.00e+00   4.89e-06   1.96e-01  60.7%
Combinational          2.93e-14   7.99e-15   5.93e-06   5.93e-06   0.0%
Clock                  6.92e-02   5.77e-02   6.48e-07   1.27e-01  39.3%
Macro                  0.00e+00   0.00e+00   0.00e+00   0.00e+00   0.0%
Pad                    0.00e+00   0.00e+00   0.00e+00   0.00e+00   0.0%
----------------------------------------------------------------
Total                  2.65e-01   5.77e-02   1.15e-05   3.23e-01 100.0%
                          82.1%      17.9%       0.0%

```

**Comment by maliberty** [comment]
My earlier suggestion was to run this in `sta` not `openroad`.  sta keeps Verilog hierarchy while OR doesn't (currently).  There is no hierarchical ces_0_0 instance in OR to annotate against.  [Technically slightly inaccurate but not in important ways].

**Comment by oharboe** [comment]
Does ORFS build sta?

**Comment by maliberty** [comment]
Yes, just type `sta`

**Comment by oharboe** [comment]
Very good. This issue is closed as such, will open a new issue if I have something more actionable...

**Comment by tspyrou** [comment]
@oharboe in results/asap7/mock-array/base/6_final.sdc the set_max_delay exception is pointing to the mock array instance which has now been flattened and does not exist in opendb. I see that you are running in openroad versus standalone sta. If you run in standalone sta this instance will not be flattened and the exception will work.

For standalone sta the script needs to remove openroad specific commands such as read_lef, utl:: and suppress_message, unsuppress_message etc. Once this is done, I see it working perfectly.

remove suppress and unsuppress from liberty.tcl

new power.tcl with openroad specific commands commented out :
#utl::set_metrics_stage "power__{}"

source $::env(SCRIPTS_DIR)/load.tcl

source $::env(SCRIPTS_DIR)/read_liberty.tcl
#read_lef $::env(TECH_LEF)
#read_lef $::env(SC_LEF)

puts "Reading results/asap7/mock-array_Element/base/6_final.v"
read_verilog results/asap7/mock-array_Element/base/6_final.v
puts "Reading $::env(RESULTS_DIR)/6_final.v"
read_verilog $::env(RESULTS_DIR)/6_final.v

link_design MockArray

read_sdc $::env(RESULTS_DIR)/6_final.sdc
# FIXME what about Element
read_spef $::env(RESULTS_DIR)/6_final.spef

report_power
read_power_activities -scope TOP/MockArrayTestbench/postSynthesis -vcd designs/src/mock-array/MockArrayTestbench.vcd
report_power
